README.md

```diff
@@ -2,41 +2,96 @@
 
 [![Build Status](https://travis-ci.org/Kinto/kinto-admin.svg)](https://travis-ci.org/Kinto/kinto-admin)
 
-A Web admin UI to manage Kinto collections.
+A Web admin UI to manage [Kinto](https://kinto.readthedocs.org/) collections.
+[Give it a go](http://kinto.github.io/kinto-admin/)!
 
 ## Installation
 
 NodeJS v4+ and npm 2.14+ should be instaled and available on your machine.
 
 ```bash
-$ npm install
+$ npm install kinto-admin --save
```

Comment:
Yes, of course :-) I was really more thinking about kinto-admin as a standalone thing, not as something that would be a dependency of your project (a lib). So maybe it's worth exchanging to have the "install globally" first, and then the "install as a dependency of your project" in the second place?